jfrog-applications/ci-and-sdks/ci-integrations/maven-artifactory-plugin.md

```diff
@@ -50,11 +50,11 @@ A typical build plugin configuration would be as follows:
 </build>
 ```
 
-The plugin's invocation phase is validate by default, and we recommend you don't change it so the plugin is called as early as possible in the lifecycle of your Maven build.
+The plugin's invocation phase is validated by default, and we recommend you don't change it so the plugin is called as early as possible in the lifecycle of your Maven build.
```

Comment:
The invitation phase is called "Validate". Perhaps it is best to make Validate a link to a Maven document describing the lifecycle phrases, to make it clearer this isn't a phrasing problem.